components/layout.js

```diff
@@ -16,8 +16,21 @@ export default function Layout({ subtitle, title, children }) {
                 </Link>
                 <div>
                     <h1 className="font-rubik font-bold text-6xl">{title}</h1>
-                    <h3 className="font-rubik text-2xl">{subtitle}</h3>
+                    {subtitle && <h3 className="font-rubik text-2xl">{subtitle}</h3>}
                 </div>
+                {author && (
+                    <div className="flex flex-row align-middle space-x-3">
+                        <img src={authorPfp} alt={authorName} className="w-14 h-14 rounded-full" />
+                        <div className="-space-y-1">
+                            <p className="font-mono font-medium text-2xl">{authorName}</p>
+                            <p className="font-rubik font-light text-lg">{publishDate}</p>
+                        </div>
+                    </div>
+                )}
+                {/* <div className="flex flex-row">
```

Comment:
do we need it to be commented?

Reply:
oh oops i was trying different things and i forgot to delete it